src/main/cpp/commands/IntakeOn.cpp

```diff
@@ -0,0 +1,34 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#include "commands/IntakeOn.h"
+
+IntakeOn::IntakeOn(double speed, Intake& intake)
```

Comment:
Subsystem goes first. And you'll have to change the initializer list order.

```suggestion
IntakeOn::IntakeOn(Intake& intake, double speed)
```